src/target/adiv5.c

```diff
@@ -945,6 +1002,14 @@ void adiv5_dp_init(adiv5_debug_port_s *const dp)
 	/* Probe for APs on this DP */
 	size_t invalid_aps = 0;
 	dp->refcnt++;
+
+	if (dp->target_designer_code == JEP106_MANUFACTURER_FREESCALE)
```

Comment:
This outer `if` needs its braces added as the inner `if` being a multi-line block makes this a multi-line block too (technically this is valid C, but we generally consider this confusing  and try to avoid doing it).